Makefile

```diff
@@ -86,13 +87,19 @@ vet:
 generate: controller-gen
 	cd api; $(CONTROLLER_GEN) object:headerFile="../hack/boilerplate.go.txt" paths="./..."
 
+comma := ,
+
 # Build the docker image
 docker-build:
-	docker build . -t ${IMG}
+ifneq (,$(findstring $(comma),$(IMG_ARCHS)))
+	@echo "docker buildx build --load currently works with one one platform"
+	@exit 2
+endif
+	docker buildx build . -t ${IMG} --platform=${IMG_ARCHS}  --load
```

Comment:
Given the `Makefile` is mostly aimed at making someone's life easier while developing, having a default that errors is a no-go for me.

Reply:
The issue with this is that new contributors need to figure out `buildx`. The Dockerfile is no longer compatible with `docker build` due to `FROM --platform`.

Reply:
On my (Arch Linux) machine which was recently installed and has a vanilla Docker config, `make docker-build IMG_ARCHS=linux/amd64` runs without any configuration tweaks.